.maintain/frame-weight-template.hbs

```diff
@@ -53,11 +53,8 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		//  Measured:  `{{benchmark.base_recorded_proof_size}}{{#each benchmark.component_recorded_proof_size as |cp|}} + {{cp.name}} * ({{cp.slope}} ±{{underscore cp.error}}){{/each}}`
 		//  Estimated: `{{benchmark.base_calculated_proof_size}}{{#each benchmark.component_calculated_proof_size as |cp|}} + {{cp.name}} * ({{cp.slope}} ±{{underscore cp.error}}){{/each}}`
 		// Minimum execution time: {{underscore benchmark.min_execution_time}} nanoseconds.
-		{{#if (ne benchmark.base_calculated_proof_size "0")}}
-		Weight::from_parts({{underscore benchmark.base_weight}}, {{benchmark.base_calculated_proof_size}})
-		{{else}}
 		Weight::from_ref_time({{underscore benchmark.base_weight}})
-		{{/if}}
```

Comment:
This will always add `0`. Maybe keep the if?

Reply:
i think adding 0 might not be bad? We assume most things will have a base PoV size, and if it doesnt, this can explicitly call it out. I mean i dont really have a strong opinion, but could make it clear that a PoV WAS calculated to be zero, versus not sure if it was calculated.